src/Components/Articles.jsx

```diff
@@ -0,0 +1,52 @@
+import ArticleCard from "./ArticleCard"
+import { useEffect, useState } from "react";
+
+
+
+function Articles() {
+    const [articles, setArticles] = useState([])
+    const [isError, setIsError] = useState(false);
+    const [isLoading, setIsLoading] = useState(false);
+    const articleURL = "https://be-nc-news-yg12.onrender.com/api/articles"
+
+    useEffect(() => {
+        setIsLoading(true)
+        fetch(articleURL)
+            .then((response) => {
+                if (!response.ok) {
+                    return Promise.reject({ status: 404, msg: "404 Not Found"})
+                }
+                return response.json()
```

Comment:
It would be better to separate the api call into another file. And also axios is recommended rather than fetch. It'll help with error handling as well